Source/carthage/Extensions.swift

```diff
@@ -108,6 +109,7 @@ internal func waitOnCommand<T>(producer: SignalProducer<T, CommandError>) -> Res
 		|> then(SignalProducer<(), CommandError>.empty)
 		|> wait
 	
+	TaskDescription.waitForAllTaskTermination()
```

Comment:
I can't determined where I should place `dispatch_group_wait()` on another place.
RAC's `completed` event timing is not fit for place it.